docs/dev/developer-guides/bridging/l1-l2.md

```diff
@@ -7,7 +7,7 @@ This section describes the interface for interaction with zkSync from L1. It ass
 
 ::: warning
 
-Please note that in the new `0.13.0-beta.2` SDK we changed the API to rename ergs to gas, to avoid breaking changes in your integration, update all your project dependencies.
+Please note that in the new `0.13.0-beta.2` SDK the API layer operates with gas. The ergs consept is used by VM only.
```

Comment:
Please change it in all other places too

```suggestion
Please note that in the new `0.13.0-beta.2` SDK the API layer operates with gas. The ergs concept is used by VM only.
```

Reply:
fixed